.github/styles.css

```diff
@@ -0,0 +1,142 @@
+.confused{
+    width:50%;
+    display: block;
+    margin: auto;
+    padding: 2%;
+}
+
+header{
+    font-size: 25px;
+    display: flex;
+    justify-content: center;
+    color: white;
+}
+p{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: x-large;
+    text-align: center;
+    color: white;
+}
+body{
+    background-color: black;
+}
+
+#bio{
+    display: flex;
+    justify-content: flex-end;
+}
+
+.container{
+position: absolute;
+  color: white;
+  top: 50%;
+  bottom: 50%;
+  
+}
+.birria{
+    width:170px;
+    height: 170px;
+}
+.scampi{
+    width:170px;
+    height: 170px;
+}
+.strawberry{
+    width:170px;
+    height: 170px;
+}
+.chocolate{
+    width:150px;
+    height: 160px;
+}
+.img-section{
+    display: flex;
+    justify-content: space-between;
+    border: solid white;
+    position: relative
+}
+h1{
+    color: white;
+}
+.dog{
+    width:170px;
+    height: 170px;  
+}
+.art{
+    width:170px;
+    height: 170px;
+}
+.Miami{
+   width: 170px;
+   height: 170PX;
+}
+.extras{
+    display: flex;
+    justify-content: space-evenly;
+    margin-top: 20%;
+    border: solid white;
+}
+h2{
+    color: white;
+    text-align:center;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
consider sticking with one (mayyybe two) font choices throughout the site to maintain a consistent look and feel of your design system!